liner.go

```diff
@@ -54,3 +63,55 @@ func (cl *contLiner) Accepted() {
 	cl.State.AppendHistory(cl.buffer)
 	cl.buffer = ""
 }
+
+func (cl *contLiner) Reindent() {
+	var min, max int
+	newDepth := cl.countDepth()
+
+	if newDepth > cl.depth {
+		min, max = cl.depth, newDepth
+	} else {
+		min, max = newDepth, cl.depth
+	}
+
+	lines := strings.Split(cl.buffer, "\n")
+	lastLine := lines[len(lines)-1]
+
+	cursorUp()
+	if len(lines) > 1 {
+		fmt.Printf("\r%s", promptContinue)
+	} else {
+		fmt.Printf("\r%s", promptDefault)
+	}
+
+	cl.printIndent(min)
+	fmt.Print(lastLine)
+	cl.printIndent(max - min)
```

Comment:
You're right. I changed it to use Ctrl-K and ported it for windows.  2203e97
I confirmed it works on windows again.